test/tool/MainEditorDatabaseTool.re

```diff
@@ -1,10 +1,21 @@
 type window;
 
-[@bs.val] external window : Js.t({..}) = "global";
+[@bs.val] external window: Js.t({..}) = "global";
 
 let buildFakeLocalStorage = () => {
   let fakeLocalStorage = WonderCommonlib.MutableHashMapService.createEmpty();
-  window##localStorage#=fakeLocalStorage
+
+  window##_localStorage #= fakeLocalStorage;
+};
+
+let removeFakeLocalStorageItem = key => {
+  let localStorage =
+    WonderCommonlib.MutableHashMapService.deleteVal(
+      key,
+      window##localStorage,
+    );
+
+  window##localStorage #= localStorage;
```

Comment:
localStorage？ use _localStorage???

Reply:
yes, window Object localStorage is _localStorage